addons/stock/models/res_config_settings.py

```diff
@@ -92,3 +92,11 @@ def set_values(self):
                 ('delivery_steps', '=', 'ship_only')])
             active = False
         warehouses.mapped('int_type_id').write({'active': active})
+
+    def execute(self):
+        res = super(ResConfigSettings, self).execute()
+        self.ensure_one()
+        if self.group_stock_multi_locations or self.group_stock_production_lot:
+            self.env['stock.picking.type'].search([('show_operations', '=', False)]
+                                                  ).write({'show_operations': True})
```

Comment:
if you tick delivery packages you also want to enable this
also i prefer if you write
``` python
picking_types = self.env['stock.picking.type'].search([('show_operations', '=', False)]
picking_types.write({'show_operations': True})
```
maybe you also want to enable the settings on the archived picking types (pick/pack etc are already created but archived)

Reply:
It's done

Reply:
last question: move_line_nosuggest_ids field is stil defined, is it still useful?

Reply:
Indeed, I removed it